recipes/deepforest/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "deepforest" %}
+{% set version = "0.2.19" %}
+{% set sha256 = "d2121897d6e32db577326f45871990721d835de38805e0547ebaa2ebe4b28dce" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win or py<36]
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip >=20.1
+    - numpy >=1.14.0
+    - cython >=0.28
+
+  run:
+    - python
+    - matplotlib-base
+    - sphinx
+    - recommonmark
+    - pyyaml
+    - pytest
+    - numpydoc
+    - cython >=0.28
+    - h5py
+    - xmltodict
+    - tqdm
+    - numpy >=1.14.0
```

Comment:
Please see https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
@scopatz thanks for the quick feedback, changed the run section 
```- {{ pin_compatible('numpy') }}``` and passed tests.